locales/pt_BR/pt_BR.json

```diff
@@ -654,7 +654,7 @@
     "United States Minor Outlying Islands": "Ilhas Menores Distantes dos Estados Unidos",
     "United States Outlying Islands": "Ilhas Distantes dos Estados Unidos",
     "Update": "Atualizar",
-    "Update & Continue Editing": "Atualizar E Continuar A Edição",
+    "Update & Continue Editing": "Atualizar e Continuar \ Edição",
```

Comment:
Are you sure you need to escape the space character?

Reply:
Sorry for the fat finger. The correct is "Update & Continue Editing": "Atualizar e Continuar a Edição"
I will fix it now.